Automattic/newspack-theme
# feat: add WhatsApp icon

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a WhatsApp icon to the Social Links menu.

See 1200550061930446-as-1205811732634485

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Add a link to a WhatsApp channel to the Social Links menu (eg. https://whatsapp.com/channel/0029VaCzzD5It5rxtjfrj71U).
3. View on the front-end and confirm that the WhatsApp icon appears. 
4. If your header has a white background, swap it a dark background (or vis versa) to confirm the SVG swaps to the correct contrast. 

![image](https://github.com/Automattic/newspack-theme/assets/177561/575befc6-09a4-4aad-a0b0-75e483db02d8)


### Other information:
